src/__tests__/modules/EditProfileModule/Section/JobInfo.test.tsx

```
@@ -97,4 +55,185 @@ describe('JobInfoSection', () => {
+  it('handles skill change', () => {
+    fireEvent.change(screen.getByLabelText('Keahlian *'), {
+      target: { value: 'Konstruksi' },
+    });
+    expect(mockOnChange).toHaveBeenCalledWith({ skill: 'Konstruksi' });
+  });
+
+  it('formats price to empty string for falsy input', () => {
+    cleanup();
+    render(
+      <JobInfoSection
+        data={{ ...mockUserProfile, price: null as any }}
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)